ARCs/arc-0036.md

```diff
@@ -0,0 +1,102 @@
+---
+arc: 36
+title: Convention for declaring filters of an NFT
+description: This is a convention for declaring filters in an NFT metadata
+author: Stéphane Barroso (@sudoweezy)
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/163
+status: Draft
+type: Standards Track
+category: ARC
+created: 2022-01-04
+requires: 16
```

Comment:
This is great, thanks for submitting @SudoWeezy. 

I am not sure about the requires on 16. Feels like filters are composable with other ARCs, e.g. ARC3 and 69. 

I guess this is a general comment on composability - something I am not too experienced with.

Reply:
totally agree with @stein-f , the new "filters" section should be optionally be able to be placed (with backward compatibility) into both ARC16 and ARC69.